oak-doc/src/site/markdown/query/lucene.md

```diff
@@ -483,14 +487,23 @@ type
   type is not used consistently across various nodes then it would recommended
   to specify the type explicitly.
   A binary is only indexed if there is an associated property `jcr:mimeType`.
+  For binary properties, you do not need to index the property separately. 
+  Binary properties are automatically added to the fulltext index (but only there), 
+  if the node is part of the index, if the node type (or mixin) matches, 
+  if the mime type property is set, and if the mime type is indexed
+  (see the Tika configuration).
```

Comment:
This would be more clear as a bullet list. Do all of the conditions have to apply for a binary property to be indexed? My understanding is that yes. So it would more clear to say this explicitly, like: `if all of the conditions below are true:`

Reply:
Done.